chain/cosmos/cosmos_chain.go

```diff
@@ -213,7 +214,7 @@ func (c *CosmosChain) RecoverKey(ctx context.Context, keyName, mnemonic string)
 
 // Implements Chain interface
 func (c *CosmosChain) GetAddress(ctx context.Context, keyName string) ([]byte, error) {
-	b32Addr, err := c.getFullNode().KeyBech32(ctx, keyName)
+	b32Addr, err := c.getFullNode().KeyBech32(ctx, keyName, "")
```

Comment:
nit: 

Having to pass a blank string is not common in Go APIs.

I've seen this cleaned up via 2 functions/methods. One could be something like `StandardKeyBech32` which only takes ctx and keyName. The other would be the `KeyBech32` and have the 3 arguments.

Reply:
Good idea, will update